Day4/ProfileComponent.js

```diff
@@ -0,0 +1,31 @@
+import { Component } from 'react';
+class ProfileComponent extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            userInfo: {}
+        };
+    }
+
+    render() {
+        const {login,company,avatar_url} = this.state.userInfo;
+        return (
+            <div>
+                <img src={avatar_url}/>
+                <p>{login}</p>
+                <p>{company}</p>
+            </div>
+        );
+    }
+
+    async componentDidMount(){
```

Comment:
You can keep all the functions before the render function - just for better readability.

Reply:
Resolved